### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by carolinepacheco**
Some text improvements

### CODE REVIEW

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 1]
```diff
@@ -4332,11 +4332,10 @@ A demo video of these workflows is available in https://www.youtube.com/user/Act
 
 === Satellite Imagery Bucket
 
-*Satellite Imagery Bucket* searching, downloading and retrieving the metadata of Sentinel satellite
-images.
+The `satellite-imagery` bucket contains some tasks that enable you search, download and retrieve the metadata of Sentinel satellite images from the https://peps.cnes.fr/rocket/#/home[PEPS website^] and https://scihub.copernicus.eu/[Copernicus Open Access Hub^].
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: enable you to ...

Comment (mklkun) [inline src/docs/MLOS/MLOSUserGuide.adoc]: that enable you search => that enable search

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 2]
```diff
@@ -4347,10 +4346,10 @@ image::Fetch_Satellite_Images.png[align=center]
 |===
 | *Variable name* | *Description* | *Type*
 | `HOST_SITES`
-| Specify a sentinel product exploitation platform.
-| List [Copernicus, Peps, All] (default=Peps)
+| Defines a utility to search and download satellite images.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: a => an

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 3]
```diff
@@ -4359,7 +4358,7 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
 
 ==== Fetch_Satellite_Images_From_PEPS
 
-*Task Overview:* Load and return a PEPS dataset containing a folder with the metadata and a folder with the downloaded images.
+*Task Overview:* Load and return a PEPS dataset containing a `metadata folder` containing metadata files and `images folder` containing satellite images.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: refactor this because there is a repetition of containing

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 4]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
 |===
 | *Variable name* | *Description* | *Type*
 | `LOCATION`
-|  Town name.
+|  Defines a town name.
 | String (default=Indonesia)
-| `COLLECTION`
-| Collection within theia collections.
+| `PLATFORM_NAME`
+| Specifies instrument on a Sentinel satellite.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: an instrument or instruments

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 5]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
...
 | List [S1, S2, S2ST, S3] (default=S2)
 | `PRODUCT_TYPE`
-|  Product type.
+|  Limits search to a Sentinel product type.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: the search

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 6]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
...
 |  List [GRD, SLC, OCN (for S1) or S2MSI1C S2MSI2A S2MSI2Ap (for S2)] (default=S2MSI1C)
 | `SENSOR_MODE`
-| Sensor mode.
+| Specifies search to a Sentinel sensor mode.
 | List [EW, IW , SM, WV (for S1) or INS-NOBS, INS-RAW (for S3)] (default=INS-NOBS)
 | `START_DATE`
-| Start date of the query in the format YYYYMMDD.
+| Starts date of the query in the format YYYYMMDD.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: to refactor as well. starts date? what does mean?

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 7]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
...
 | String
 | `END_DATE`
-| End date of the query in the format YYYYMMDD.
+| Ends date of the query in the format YYYYMMDD.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: same

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 8]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
...
-| Sentinel tile number.
+| Limits search to a tile number.
 | String
 | `LATITUDE`
-|  Latitude in decimal degrees.
+|  Determines search to a latitude in decimal degrees.
 |  Float
 | `LONGITUDE`
-|  Longitude in decimal degrees.
+|  Limits search to a longitude in decimal degrees.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: same for this and 2 previous ones

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 9]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
...
 |  Float
 | `OUTPUT_PATH`
-|  Address path to save downloaded images.
+|  Specifies an address path to save downloaded images.
 |  String
 |===
 
-WARNING: Please add two third party credentials (USER_NAME_PEPS and USER_PASS_PEPS) in the `Scheduling and Orchestration interface -> Manage Third-Party Credentials` to connect to PEPS platform.
+WARNING: Please add third party credentials (USER_NAME_PEPS and USER_PASS_PEPS) in the `Scheduling & Orchestration interface -> Manage Third-Party Credentials` to connect to PEPS.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: You cannot use PA:Credential?

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 10]
```diff
@@ -4368,44 +4367,44 @@ NOTE: Please access link:https://peps.cnes.fr/rocket/#/register[Peps] and link:h
...
 
 NOTE: More information about the source of this task can be found https://github.com/olivierhagolle/maja_peps[here^].
 
 ==== Fetch_Satellite_Images_From_Copernicus
 
-*Task Overview:* Load and return a PEPS dataset containing a folder with the metadata and a folder with the downloaded images according to the resolution & image band selected.
+*Task Overview:* Load and return a Copernicus dataset containing a `metadata folder` containing metadata files and `images folder` containing satellite images according to the resolution & image band selected by user.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: to reformulate to avoid repetition

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 11]
```diff
@@ -4414,32 +4413,32 @@ NOTE: More information about the source of this task can be found https://github
 |===
 | *Variable name* | *Description* | *Type*
 | `PLATFORM_NAME`
-| Specific instrument on a Sentinel satellite.
+|  Specifies instrument on a Sentinel satellite.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: ssame

File: src/docs/MLOS/MLOSUserGuide.adoc [hunk 12]
```diff
@@ -4414,32 +4413,32 @@ NOTE: More information about the source of this task can be found https://github
...
-| Granule dimensions for each resolution band.
+| Defines granule dimensions for each resolution band.
 |  List [10m, 20m, 60m] (default=10m)
 | `IMAGE_BAND`
-| Choose from 13 spectral bands spanning from the Visible and Near Infra-Red (VNIR) to the Short Wave Infra-Red (SWIR).
+|  Determines from 13 spectral bands spanning from the Visible and Near Infra-Red (VNIR) to the Short Wave Infra-Red (SWIR).
 |  List [All, B01, B02, B03, B04, B05, B06, B07, B08, B8A, B09, B10, B11, B12, TCI] (default=All)
 | `OUTPUT_PATH`
-| Address path to save downloaded images.
+| Specifies an address path to save downloaded images.
```
Comment (mboussaa) [inline src/docs/MLOS/MLOSUserGuide.adoc]: same for all

**Pull request merged**